source/common/tracing/zipkin_tracer_impl.cc

```diff
@@ -0,0 +1,201 @@
+#include "common/tracing/zipkin_tracer_impl.h"
```

Comment:
Can we move this file into the zipkin directory and the zipkin namespace. There really isn't any "library" anymore so I see no reason to have it be outside that namespace.